components/list/list-item-drag-mixin.js

```diff
@@ -0,0 +1,498 @@
+import { css, html } from 'lit-element/lit-element.js';
+import { announce } from '../../helpers/announce.js';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { dragActions } from './list-item-drag-handle.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { nothing } from 'lit-html';
+
+export const dropLocation = Object.freeze({
+	above: 1,
+	below: 2,
+	first: 3,
+	last: 4,
+	shiftDown: 5,
+	shiftUp: 6,
+	void: 0
+});
+
+const dropTargetLeaveDelay = 1000; //ms
+
+export const ListItemDragDropMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			/**
+			 * Whether the item is draggable
+			 */
+			draggable: { type: Boolean, reflect: true },
+			dragging: { type: Boolean, reflect: true,  },
```

Comment:
Dangling comma here.

Also, is this meant to be private?  If so, we should prefix is with underscore so it's clear that consumers are not meant to touch it.

Reply:
It is a read only style one. So making it private would be a good idea. Then set the `attribute: 'dragging'` for it to still work